### PR TITLE
feat: Use same icons in page tree state indicators and Manage verisons

### PR DESCRIPTION
**Pull request by fsbraun**
## Description

This PR reuses the icons of the django CMS core (which are loaded anyways) for the "Manage Verions" admin view. This creates a more uniform look and feel.

**Before (with djangocms-versioning icons):**
Bildschirmfoto 2022-12-15 um 09 26 22

**After (with django-cms core icons):**
Bildschirmfoto 2022-12-15 um 09 27 59

## Related resources: icons of django CMS core
Bildschirmfoto 2022-12-15 um 09 32 01

* The rules for showing the icons are identical to those of #295 ensuring a consistent experience: Icon is shown if the workflow allows for the action. If the user cannot execute it is grayed out.
* All icons are now produced using the `templates/djangocms_versioning/admin/icons/` templates (before some icons had a independent templates)
* Works django admin (i.e., without djangocms-admin-style) and djangocms-admin-style 3.2.1+ (https://github.com/django-cms/djangocms-admin-style/pull/477); earlier versions inject css poisoning icon usage
## Checklist

* [x] I have opened this pull request against ``master``
* [x] I have added or modified the tests when changing logic
* [x] I have followed [the conventional commits guidelines](https://www.conventionalcommits.org/) to add meaningful information into the changelog
* [x] I have read the [contribution guidelines ](https://github.com/django-cms/django-cms/blob/develop/CONTRIBUTING.rst) and I have joined #workgroup-pr-review on 
[Slack](https://www.django-cms.org/slack) to find a “pr review buddy” who is going to review my pull request.

### CODE REVIEW

File: djangocms_versioning/templates/djangocms_versioning/admin/icons/view.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{% extends "./base.html" %}
+{% load static i18n %}
```
Comment (marksweb) [inline djangocms_versioning/templates/djangocms_versioning/admin/icons/view.html:2]: Can drop static

File: djangocms_versioning/templates/djangocms_versioning/admin/icons/unpublish_icon.html [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{% extends "./base.html" %}
+{% load i18n %}
+{% block title %}{% trans 'Unpublish' %}{% endblock %}
```
[the remark this answers is not in the document]
Comment (fsbraun) [reply djangocms_versioning/templates/djangocms_versioning/admin/icons/unpublish_icon.html:3]: @marksweb You mean as opposed to `{% translate %}`?  Did not change that but I guess it's heritage from old Django.

[the remark this answers is not in the document]
Comment (marksweb) [reply djangocms_versioning/templates/djangocms_versioning/admin/icons/unpublish_icon.html]: That's right. But if we drop old Django then running django-upgrade may fix these for us.

[the remark this answers is not in the document]
Comment (fsbraun) [reply djangocms_versioning/templates/djangocms_versioning/admin/icons/unpublish_icon.html]: I fixed this here. But django-upgrade will find many more... :-)

**Review by marksweb — APPROVED** [review]
Thanks 👏

**Pull request merged**